app/versions/operational-reporting/v3/components/search-box/init.js

```diff
@@ -0,0 +1,56 @@
+const v3SearchBox = document.querySelector('#v3-search-box')
```

Comment:
A question for my own understanding mainly. 
Why is there a v2 and v3 version of the search?

Reply:
I demo'd v2 of the search (which is a prompted version) to Derrick, and he said he also wanted to see a simple version that searched the attributes as text. You can see version 2 here: https://digital-prison-reporting-mi-prototype-main.apps.live.cloud-platform.service.justice.gov.uk/operational-reporting/

All the JS files end up on the same layout, so the ID (and CSS class on the page) is used to differentiate between the different versions.